### ISSUE TITLE
Plugin does not work

### ISSUE THREAD

**Issue by zentale** [etopeter/kubectl-view-utilization#16]
Hi, I am using ubuntu 16.04, with the below versions of the different components. The plugin does not work in this setup.
```
$ bash --version
GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

$ awk -W version
mawk 1.3.3 Nov 1996, Copyright (C) Michael D. Brennan

compiled limits:
max NF             32767
sprintf buffer      2040

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.2", GitCommit:"cff46ab41ff0bb44d8584413b598ad8360ec1def", GitTreeState:"clean", BuildDate:"2019-01-10T23:35:51Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.8", GitCommit:"7eab6a49736cc7b01869a15f9f05dc5b49efb9fc", GitTreeState:"clean", BuildDate:"2018-09-14T15:54:20Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
$  kubectl krew install view-utilization
Updated the local copy of plugin index.
Installing plugin: view-utilization
W0128 15:49:33.278503   16523 install.go:128] Skipping plugin view-utilization, it is already installed
$ kubectl view-utilization
awk: line 13: syntax error at or near *
awk: line 14: syntax error at or near *
awk: line 15: syntax error at or near *
awk: line 16: syntax error at or near *
awk: line 16: extra ')'
```

**Comment by etopeter** [comment]
I need to investigate compatibility of mawk over awk or gawk. Mawk is meant to be faster but it's stripped out of features therefore I would recommend switching to awk or gawk. Plugin could either be written in a way to support all of them or do compatibility check before executing.

You can switch default awk in Ubuntu by running
`sudo update-alternatives --config awk`

**Comment by etopeter** [comment]
The fix for this issue is in latest release v0.1.3. Please check if its resolved in your environment so we can close this issue.

**Comment by etopeter** [comment]
Due to inactivity I'm going to close this issue. Please reopen if your this issue still exists.
